dbld/images/helpers/packages.manifest

```diff
@@ -238,10 +238,14 @@ gdb                             [devshell]
 gdbserver                       [devshell]
 joe                             [devshell]
 lcov                            [devshell]
+libbson-1.0-0=1.15.0-1          [devshell]
+libbson-dev=1.15.0-1            [devshell]
 libc6-dbg                       [devshell]
```

Comment:
why do you need specific versions from these?

Reply:
These are the exact versions in OBS:

https://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_18.04/amd64/libbson-1.0-0_1.15.0-1_amd64.deb.mirrorlist

I set the specific version, to make sure that that is the one installed.

Reply:
But this means that whenever Laci updates the packages there our image building would break.

Reply:
Oh, I did not think of that, you are right. Let me think of a solution. You might now the answer already, if so, I am open to your suggestion :)

Reply:
I see, that the OBS repo is set to lowest in the apt preferences:
```
/etc/apt# cat preferences.d/lbudai 
Package: *
Pin: origin "download.opensuse.org"
Pin-Priority: 1
```
If I set the `Pin-Priority` to `1001`, `apt` automatically finds the package from OBS.

But the question arises, is it a good thing to raise the OBS preference to above everything else?
What do you think of this solution?

Reply:
Apt does not support version range (for example minimal version). That would have been the best case.

We want to keep obs depencency as low as possible, so I would not raise the repository preference.

If it worths the effort, we can implement the missing feature for apt: with an apt search we can enumerate all available versions, and pick the largest.

But updating obs packages should not happen that often. In my opinion, we can live with that inconvenience.